testdata-generator-common/src/main/java/io/openepcis/testdata/generator/model/AbstractEventCreationModel.java

```diff
@@ -235,16 +235,20 @@ public List<String> referencedParentIdentifier(final List<EventIdentifierTracker
                 .findFirst()
                 .ifPresent(
                         t -> parentList.addAll(EventModelUtil.parentIdentifiers(t, epc.getInheritParentCount())));
-        epc.setInheritParentCount(0);
+        //epc.setInheritParentCount(0);
       }
```

Comment:
please remove line completely, commented code may distract from new issues showing up in the future and result in this line to be added again by mistake.